examples/bin/start-druid

```diff
@@ -0,0 +1,548 @@
+#!/usr/bin/env python3
```

Comment:
is this chill? should there be a p2 + p3 version? Have a look at how `post-index-task` does it - it is a bash script that routes to the appropriate python because we can not assume that the user has python3

Reply:
I haven't tried out running with p2, since it was end of life. I will update so that it works with both p2 and p3.

Reply:
Ah, even if we rely on Python 3, there's still another issue: sometimes people don't literally have `python3` on their path, even though python 3 is available. It's sometimes only called `python`. So this should do the same launcher trick that we have in `examples/bin/post-index-task`, where we try both.

If it's easy to support python 2 _and_ 3, please do that. It will make life easier for users on older OSes.

Either way, we should also add a system requirement for whichever Python versions we end up supporting. Interpreters aren't always present and people may need to install them. Please add this in the places that we mention system requirements:

- after "Java 8u92+" in `tutorials/index.md`
- the "Select OS" section in `tutorials/cluster.md`

As @FrankChen021 mentioned, users might also not be sure how to install a Python interpreter if they don't have one already. So it would also be helpful to describe how to install them on common OSes. For example, on Ubuntu, it can be installed by running:

```
apt-get -y update
apt-get -y install python3
```

Btw, one reason we chose perl for the `bin/supervise` script is that perl is more widely built-in to OSes than python. I guess a lot of OSes use it for their internal scripting. This is also why we don't list perl in the system requirements: it seems to really be ubiquitous. IMO, it's still ok to use python, but we should take these steps to make it easy for people to satisfy the requirement.

Reply:
Added support for python2 and python3.
Updated system requirements. 
Added steps for installing Python interpreter.